src/components/register/RegisterCardTwo.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <div class="register-card-wrapper">
+    <form @submit.prevent="submitForm" novalidate>
+      <div class="register-card-wrapper__form form-field">
+        <MainInput v-model="firstName" inputName="first-name" inputType="text"
+          >FIRST NAME</MainInput
```

Comment:
By the way, a cool, small tip for you, especially if you would like to add such strings to i18n in the future 💡

We can actually not write text in a full caps like here, but normally `First name` and then make... CSS capitalize it for us 😄

```css
text-transform: uppercase;
```

If all your inputs have the labels in all caps - we can style it as such in the `MainInput` component itself. If not, maybe add a prop to sometimes capitalize it and apply a class to do so? 🚀

Reply:
this is a good tip!!! Thank you 💡